mods/tuxemon/db/technique/overfeed.json

```diff
@@ -0,0 +1,53 @@
+{
+  "accuracy": 1,
+  "animation": "drip_green",
+  "can_apply_status": true,
+  "category": "physical",
+  "effects": [
+    "statchange"
+  ],
+  "healing_power": 0,
+  "icon": "",
+  "is_area": false,
+  "is_fast": false,
+  "potency": 1,
+  "power": 0,
+  "statspeed": {
+    "value": 2, "dividing":true, "overridetofull":false
```

Comment:
lets document these values somewhere.  im not familiar with this, and its not clear what "dividing", and "overridetofull" mean.  my suggestion would be to add the documentation to the function which uses these values.

Reply:
ok, ill do that in the technique.py